wallet/AndroidManifest.xml

```diff
@@ -233,6 +233,15 @@
             android:screenOrientation="portrait"
             android:theme="@style/My.Theme.ChildActivity" />
 
+        <activity android:name="de.schildbach.wallet.ui.ReceiveActivity"
+            android:screenOrientation="portrait"
+            android:theme="@style/My.Theme.ChildActivity" />
+
+        <activity
+            android:name="de.schildbach.wallet.ui.TransactionResultActivity"
+            android:screenOrientation="portrait"
+            android:theme="@style/My.Theme.Fullscreen" />
+
```

Comment:
Seems like something isn't working correctly with GitHub's diff.  This code was added in the Pay/Receive PR and it is already in the `redesign` branch.

Reply:
Nevermind on this.